assets/wizards/advertising/views/ad-units/index.js

```diff
@@ -36,8 +36,8 @@ const AdUnits = ( {
 	serviceData,
 	fetchAdvertisingData,
 } ) => {
-	const gamConnectionErrorMessage = serviceData?.status?.error
-		? `${ __( 'Google Ad Manager connection issue', 'newspack' ) }: ${ serviceData.status.error }`
+	const gamErrorMessage = serviceData?.status?.error
+		? `${ __( 'Google Ad Manager Error:', 'newspack' ) }: ${ serviceData.status.error }`
```

Comment:
There's already a colon in this message.

```suggestion
		? `${ __( 'Google Ad Manager Error', 'newspack' ) }: ${ serviceData.status.error }`
```